app/dashboard/models.py

```diff
@@ -3245,7 +3256,9 @@ class HackathonEvent(SuperModel):
     logo_svg = models.FileField(blank=True)
     start_date = models.DateTimeField()
     end_date = models.DateTimeField()
-    identifier = models.CharField(max_length=255, default='')
+    background_color = models.CharField(max_length=255, null=True, blank=True, help_text='hexcode for the banner')
+    identifier = models.CharField(max_length=255, default='', help_text='used for custom styling for the banner')
```

Comment:
blank=True to make it not required?

Reply:
default = '' would never make it blank :P